save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/contests/ContestSummaryMenu.kt

```diff
@@ -30,6 +31,72 @@ external interface ContestSummaryMenuProps : Props {
     var contestName: String
 }
 
+private fun ChildrenBuilder.displayTopProjects(results: List<ContestResult>) {
+    ul {
+        className = ClassName("col-10 mb-2 list-group")
+        displayResult(
+            "Top position",
+            "Project",
+            "Organization",
+            "Score",
+            null
+        )
+        results.filter {
```

Comment:
It's too implicit that results here are already sorted (also - are they being sorted twice, both on backend and on frontend?). Maybe rename `results` -> `resultsSorted`?

Reply:
Oh ok, will rename it

Reply:
Done